psydac/api/ast/nodes.py

```diff
@@ -1377,6 +1377,7 @@ def get_geometry_stmts(self, mapping):
 
         if mapping._expressions is not None:
             args += [ComputeLogical(WeightedVolumeQuadrature(l_quad))]
+            args += [ComputeLogical(SymbolicWeightedVolume(mapping))]
```

Comment:
What is the `SymbolicWeightedVolume`?

Reply:
it's the symbolic representation of the the weighted volume of a quadrature rule
`wvol = quad_1[k1]*quad_2[k2]` where `SymbolicWeightedVolume` is `wvol`

Reply:
Isn't that a `WeightedVolumeQuadrature` instead?

Reply:
this is `quad_1[k1]*quad_2[k2]` the `WeightedVolumeQuadrature`

Reply:
Sorry, I cannot understand. Can you write the 2 expressions of `WeightedVolumeQuadrature` and `SymbolicWeightedQuadrature` again?

Reply:
not both the `WeightedVolumeQuadrature` is used once in those lines but `SymbolicWeightedQuadrature` is used in other places for example we  can have something like
`wvol = quad_1[k1]*quad_2[k2]`
and then use it in a expression like
`v += wvol*(u_x*v_x + u_y*v_y)`